shenyu-plugin/shenyu-plugin-api/src/main/java/org/apache/shenyu/plugin/api/utils/RequestUrlUtils.java

```diff
@@ -43,7 +43,7 @@ public static URI buildRequestUri(final ServerWebExchange exchange, final String
         String path = domain;
         final String rewriteUri = (String) exchange.getAttributes().get(Constants.REWRITE_URI);
         if (StringUtils.isNoneBlank(rewriteUri)) {
-            path = path + rewriteUri;
+            path = rewriteUri;
```

Comment:
![image](https://github.com/apache/shenyu/assets/38336731/984cf8b1-33b7-4927-bf91-25939b1d1840)

This line change shall rollback.